semantic-conventions/README.md

```diff
@@ -92,11 +92,10 @@ semantic conventions that have the tag `network`.
 
 The image supports [Jinja](https://jinja.palletsprojects.com/en/2.11.x/) templates to generate code from the models.
 
-For example, opentelemetry-java [generates typed constants for semantic conventions](https://github.com/open-telemetry/opentelemetry-java/blob/main/semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java)
-using [this template](https://github.com/open-telemetry/opentelemetry-java/blob/main/buildscripts/semantic-convention/templates/SemanticAttributes.java.j2).
+For example, opentelemetry-java [generates typed constants for semantic conventions]. Refer to https://github.com/open-telemetry/semantic-conventions-java for all semantic conventions. 
 
 The commands used to generate that are
-[here in the opentelemetry-java repo](https://github.com/open-telemetry/opentelemetry-java/blob/main/buildscripts/semantic-convention/generate.sh).
```

Comment:
```suggestion
The workflows and templates used to generate the constants can also be found there.
```
Since you link to the repo (different name btw) above.